tweetapi/authentication.py

```diff
@@ -17,8 +17,11 @@ def authenticate(self, request):
 
         try:
             payload = jwt.decode(token,"secret",algorithms=["HS256"])
-            username= payload['username']
-            user = User.objects.get(username=username)
+            id= payload['id']
```

Comment:
You don't need this on a separate line. is there a chance that id is not in payload